sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -245,7 +263,19 @@ private Builder(String[] args, boolean validation,
      */
     public Builder fromArgs(String... args) {
       checkNotNull(args, "Arguments should not be null.");
-      return new Builder(args, validation, strictParsing, true);
+      options.add(current -> parseCommandLine(args, current.strictParsing));
```

Comment:
It is important to point out that this is a subtle change in behavior. Before fromArgs replaced the existing set of args and now we are accumulating them. Please update the javadoc accordingly.